### DISCUSSION TITLE
"Favorite plays" category on user profiles (for the osu! website)

### DISCUSSION THREAD

**Discussion by dukkline** [ppy/osu#13882] [Ideas]
This would consist of a section in someone's profile which could show up to 5 (more or less, you can decide what is reasonable) user-picked scores **submitted by them**, to show on their profile. It could be put either in the "Ranks" category or the "Historical" category, whatever works best for the integrity of the website.

The reason for this to exist would be to showcase plays someone set that they consider impressive and are proud of, but that don't show up on their top plays because the ammount of pp they give is low or null (for example, a pass on a high star map, or even just any score from them that they wish could be given more attention).

These scores could be presented the way they are in the "Recent Plays (24h)" section, excluding the "weighted __%" part and the weighed pp, only showing the other things (depicted in picture below). I believe this is the best way to do it, but there's a lot of things i might not have taken into consideration, and if you can think of something better, that's great.

Below is an ultra scuffed depiction of what this could look like. It's really bad, sorry, but i hope it can get the idea across. And if you have any more questions, please ask, i'll answer them the best i can

![20210713_204959](https://user-images.githubusercontent.com/84641071/125545962-6dc53fc3-5b06-4b55-a298-5f81d1fe3eaf.jpg)

**Comment by smoogipoo** [comment]
Needs an osu-web implementation first. Make an issue there: https://github.com/ppy/osu-web

**Reply by dukkline** [reply]
oh ok thank you
